code/modules/mob/living/carbon/alien/humanoid/humanoid_defense.dm

```diff
@@ -1,3 +1,18 @@
+/mob/living/carbon/alien/humanoid/hitby(atom/movable/AM, skipcatch = 0, hitpush = 0, blocked = 0)
+	hitpush = 0
+	var/obj/item/I
+	if(istype(AM, /obj/item))
+		I = AM
+	if(I)
```

Comment:
This should be better composed with `&&` so it's not a mountain of tabs and if statements